Create ui-c8y-1021-73-7-no-validation-when-wrong-hour-or-minute-is-in-date-time.md (#3316)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-73-7-no-validation-when-wrong-hour-or-minute-is-in-date-time.md

---
date: ""

> title: The date and time picker no longer accepts invalid hour or minute values

Reviewer: ```suggestion
title: Date-time picker automatically corrects invalid hour and minute values
```

> title: No validation when wrong hour or minute is in date time picker (#8829) [GRAFT][release/cd] (#8949)

Reviewer 2: ```suggestion
title: The date and time picker no longer accepts invalid hour or minute values
```

> product_area: Device management & connectivity

Reviewer: ```suggestion
product_area: Application enablement & solutions
```

change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:

>   - value: component--KIsStyzM

Reviewer: Please also change the value. And move the file into the right folder (application-enablement).

Reviewer 2: Done

>     label: Device Management app

Reviewer 2: ```suggestion
    label: Web SDK
```

>     label: Device Management app

Reviewer: Isn´t this a Web SDK change instead of Device Management app?

Reviewer 2: Done

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61596
version: 1021.73.7
---

> The date and time picker component now automatically corrects invalid hour and minute values to 00.

Reviewer: ```suggestion
The date-time picker component now automatically corrects invalid hour and minute values to "00".
```

> No validation when wrong hour or minute is in date time picker (#8829) [GRAFT][release/cd] (#8949)

Reviewer 2: ```suggestion
The date and time picker component now automatically corrects invalid hour and minute values to 00.
```
